nav2_bt_navigator/src/bt_navigator.cpp

```diff
@@ -64,6 +65,7 @@ BtNavigator::BtNavigator()
   declare_parameter("transform_tolerance", rclcpp::ParameterValue(0.1));
   declare_parameter("global_frame", std::string("map"));
   declare_parameter("robot_base_frame", std::string("base_link"));
+  declare_parameter("odom_topic", std::string("odom"));
```

Comment:
This param also needs to be added to the MD file in Nav2 and the website parameter file for the bt navigator.